src/bento/siteWideConfig.js

```diff
@@ -6,3 +6,6 @@ export default {
   // List for options for authentication empty array defaults to google
   authEndPoint: ['google'], // authEndPoint: []
 };
+
```

Comment:
can you update the branch

Reply:
Automatically updated. Pointing to Dev now.